app/controllers/proof_controller.rb

```diff
@@ -5,28 +5,25 @@ class ProofController < ApplicationController
 #  impressionist # log actions for pageview stats
 
   def create
-    if params['manifestation'].nil? # legacy site's buttons.js hack
-      unless params['what'].nil? or params['what'].empty? # don't bother capturing null submissions
-        #if is_blacklisted_ip(client_ip) # filter out spam
-        #  render plain: "OK"
-        #else
-          @p = Proof.new(:from => params['email'], :about => params['about'] || request.env["HTTP_REFERER"] || 'none', :what => params['what'], :subscribe => (params['subscribe'] == "yes" ? true : false), :status => 'new')
-          h = HtmlFile.find_by_url(@p.about.sub(/https?:\/\/.*benyehuda.org\//, ''))
-          @p.html_file = h unless h.nil?
-          @p.save!
-        #end
-      else
-        render plain: "OK"
-      end
-    else # new BYBE
-      if params['ziburit'] =~ /ביאליק/
-        @p = Proof.new(from: params['from'], manifestation_id: params['manifestation'].to_i, what: params['what'], highlight: params['highlight'], status: 'new')
-        @p.save!
-      end
+    @errors = []
+    unless params[:ziburit] =~ /ביאליק/
+      @errors << I18n.t('proof.create.ziburit_failed')
     end
-    respond_to do |fmt|
-      fmt.html { }
-      fmt.js { flash[:notice] = I18n.t(:proof_thanks_html) }
+    if params[:from].blank?
```

Comment:
Actually as far as I see Proof form says that email is optional, so perhaps we need to update wording there too.

Reply:
True! I am fixing the string and will commit after merging this.